### ISSUE TITLE
gitlab integration:  undefined method `commit_line' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by JohnBat26** [mmozuras/pronto#125]
Hello.

When I run command:

```
pronto run -f gitlab 
```

I get next error:

```
/home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/pronto-0.5.3/lib/pronto/formatter/gitlab_formatter.rb:12:in `block in format': undefined method `commit_line' for nil:NilClass (NoMethodError)
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/pronto-0.5.3/lib/pronto/formatter/gitlab_formatter.rb:7:in `map'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/pronto-0.5.3/lib/pronto/formatter/gitlab_formatter.rb:7:in `format'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/pronto-0.5.3/lib/pronto.rb:43:in `run'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/pronto-0.5.3/lib/pronto/cli.rb:52:in `run'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/gems/pronto-0.5.3/bin/pronto:6:in `<top (required)>'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/bin/pronto:23:in `load'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/bin/pronto:23:in `<main>'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/bin/ruby_executable_hooks:15:in `eval'
        from /home/gitlab-runner/.rvm/gems/ruby-2.3.0@wc_2_4/bin/ruby_executable_hooks:15:in `<main>'

```

**Comment by marcosgugs** [comment]
+1

**Comment by marcosgugs** [comment]
I'm getting the same error.

**Comment by mmozuras** [comment]
@marcosgugs which version are you using? `0.7.1`?

**Comment by marcosgugs** [comment]
@mmozuras Yes, `0.7.1`!

**Comment by siddhant-mohan** [comment]
same error, again and again

**Comment by gitanshu** [comment]
Same error in pronto-0.8.0

**Comment by mmozuras** [comment]
Released a fix with [v0.8.1](https://github.com/mmozuras/pronto/releases/tag/v0.8.1).
